cpp/cmake/thirdparty/get_nvcomp.cmake

```diff
@@ -0,0 +1,40 @@
+# =============================================================================
```

Comment:
To Vyas' point above, we can move this to the `python/cmake/thirdparty` directory (may need to create the `thirdparty` portion of that path)

Then we can add a similar `include` line here:

https://github.com/rapidsai/kvikio/blob/1ab41d6fe06f18e0e8996a4451307201bee4e787/python/cmake/CMakeLists.txt#L15

Reply:
Great, will do shortly.

Reply:
Yup. Then I think the only piece that we'll need to add is a `target_link_libraries` call telling CMake to link `kvikio::nvcomp` against the C++ `nvcomp` library.

Reply:
:+1:

Reply:
I've got all of `skbuild` working locally, up to running the python and finding out that cuda has not been linked in properly to the libkvikio.so library. Trying to figure out which CMakeLists.txt and syntax to add it with.

Reply:
You need to add `nvcomp` to the [list of cython modules in `_lib/CMakeLists.txt`](https://github.com/rapidsai/kvikio/blob/branch-22.04/python/kvikio/_lib/CMakeLists.txt#L16) then I think you should be good.